lib/miq_environment.rb

```diff
@@ -94,8 +94,9 @@ def self.supports_command?(cmd)
 
       begin
         # If 'which apachectl' returns non-zero, it wasn't found
+        require "awesome_spawn"
```

Comment:
Only other thing might be moving the require outside of the begin/rescue (that's actually where the original code had it before I removed it) - https://github.com/ManageIQ/manageiq/blame/ae1b5cacb3ac65b0317fd353a11e1f217870b291/lib/miq_environment.rb#L86